app/assets/stylesheets/govuk_publishing_components/components/_highlight-boxes.scss

```diff
@@ -66,7 +66,7 @@
 
   &:active,
   &:focus {
-    color: $govuk-link-colour;
+    color: $govuk-text-colour;
```

Comment:
Unsure, because of the `:active` state, but should this be `$govuk-focus-text-colour`?

Reply:
Yep, they all (for all the fixed focus colours) should be `$govuk-focus-text-colour`

Reply:
Checked focus states for all components and made use of `$govuk-focus-text-colour` where needed in ff411ec